smart-gate/packages/client/.envsample

```diff
@@ -0,0 +1,30 @@
+# .envsample
+# ----------------------------------------
+# This file serves as a template for environment variables.
+# DO NOT commit changes made to this file.
+#
+# To use local environment variables:
+# 1. Duplicate this file.
+# 2. Rename the duplicate to .env.
+# 3. Update the .env file with your local environment settings.
+#
+# Example:
+# cp .envsample .env
+#
+# Add the .env file to your .gitignore to prevent accidental commits.
+
+VITE_CHAIN_ID=17069
+
+VITE_SMARTASSEMBLY_ID=23563565629941518662089712946553773054394767785169002865278787679356860010098
+
+# NEBULA
+VITE_GATEWAY_HTTP="https://blockchain-gateway-nebula.test.tech.evefrontier.com"
+VITE_GATEWAY_WS="wss://blockchain-gateway-nebula.test.tech.evefrontier.com"
+
+# STILLNESS
+# VITE_GATEWAY_HTTP="https://blockchain-gateway-stillness.live.tech.evefrontier.com"
+# VITE_GATEWAY_WS="wss://blockchain-gateway-stillness.live.tech.evefrontier.com"
+
+# # LOCAL
+# VITE_GATEWAY_HTTP="http://localhost:8080"
+# VITE_GATEWAY_WS="ws://localhost:8080"
```

Comment:
These values will be updated after https://github.com/projectawakening/builder-examples/pull/79 is merged in. would be good to do a merge update and follow that pattern after it's in